cave/plot/cost_over_time.py

```diff
@@ -203,7 +203,7 @@ def plot(self, rh: RunHistory, runs: List[ConfiguratorRun],
         # Tilt tick labels
         p.xaxis.major_label_orientation = 3/4
 
-        p.legend.location = "bottom_left"
+        p.legend.location = "top_right"
```

Comment:
wouldn't "best" be more fitting?

Reply:
Automatic legend placement is still an open issue in bokeh - so I figure since we don't expect any line in this area in c-o-t, upper right corner is best position. Outside the plot would also be possible.

Reply:
ok